web/src/pages/Namespace.tsx

```diff
@@ -101,6 +101,20 @@ export const NamespacePage: FC = () => {
         <NamespaceInfoPlaceholder />
       ) : (
         <>
+          { namespace === user?.login && user?.orgs && user.orgs.length > 0 && (
+          <p className="mb-0">
+            <span className="fw-bold">Organizations you belong to: {' '}
+              {user?.orgs.map((org, index) => (
+                <Fragment key={org}>
+                  {index !== 0 && ', '}
+                  <a href={`/${org}`}>
```

Comment:
Do we want to style beyond just a link? Maybe a button or something? Or a badge? Just some ideas.

Reply:
Yeah I had thought about maybe a badge or something as well